packages/contracts/contracts/SphinxManager.sol

```diff
@@ -938,11 +935,11 @@ contract SphinxManager is
 
     // TODO(docs)
     function _deploymentFailed(DeploymentState storage _deployment, uint256 _actionIndex) private {
-        activeDeploymentId = bytes32(0);
-        _deployment.status = DeploymentStatus.FAILED;
-
         emit DeploymentFailed(activeDeploymentId, _actionIndex);
         registry.announceWithData("DeploymentFailed", abi.encodePacked(activeDeploymentId));
+
+        activeDeploymentId = bytes32(0);
+        _deployment.status = DeploymentStatus.FAILED;
```

Comment:
We have to update the `activeDeploymentId` after emitting the event b/c it's included in the event.